fluss-connectors/fluss-connector-flink/src/main/java/com/alibaba/fluss/connector/flink/FlinkConnectorOptions.java

```diff
@@ -106,6 +106,15 @@ public class FlinkConnectorOptions {
                     .defaultValue(false)
                     .withDescription("Whether to ignore retract（-U/-D) record.");
 
+    public static final ConfigOption<Boolean> SINK_PRE_HASH_BY_BUCKET_KEY =
+            ConfigOptions.key("sink.pre-hash-by-bucket-key")
```

Comment:
`sink.bucket-shuffle` for simplification. Besides, it's not "hash by bucket key", but "shuffle by bucket id". Please update the description of the option.

Reply:
Please update documentation in this PR as well.

Reply:
Done!